docs/ops/metrics.md

```diff
@@ -1654,6 +1654,9 @@ Request metrics aggregated over a subset of all entities of the respective type:
   - `/jobs/metrics?jobs=D,E,F`
   - `/jobs/<jobid>/vertices/<vertexid>/subtasks/metrics?subtask=1,2,3`
 
+<span class="label label-danger">Warning</span> Metric names can contain special characters (`?`, `&`, `$`, `+`, `/`, `,`, `:`, `=`, `@`, `%`) that you need to be escape when querying metrics.
```

Comment:
```suggestion
<span class="label label-danger">Warning</span> Metric names can contain special characters (`?`, `&`, `$`, `+`, `/`, `,`, `:`, `=`, `@`, `%`) that you need to escape when querying metrics.
```

Reply:
Can we link to the rules how to escape these characters, e.g. a table of URL Escape Characters?

Reply:
I had hoped that the example gives users enough of a reference to find a table.

The only _official_(?) reference I could find for which characters to escape is this [RFC](https://tools.ietf.org/html/rfc3986#section-2.2), but it's not as readable as I would like.
[This](https://secure.n-able.com/webhelp/NC_9-1-0_SO_en/Content/SA_docs/API_Level_Integration/API_Integration_URLEncoding.html) page was the basis for which characters I explicitly listed, but it is not really consistent with the RFC.
I originally did not want to list any characters at all, but with the RFC being _eh_ and all other sources not being _official_ I couldn't find solution I really liked.

As for how to escape, it is essentially just "% + ascii_hex(character)", which I can add.

Reply:
Can we then maybe create the table for the characters you have listed?